poetry/utils/env.py

```diff
@@ -286,7 +286,13 @@ def get(self, reload=False):  # type: (bool) -> Env
                 python_minor = env["minor"]
 
         # Check if we are inside a virtualenv or not
-        in_venv = os.environ.get("VIRTUAL_ENV") is not None
+        # Conda sets CONDA_PREFIX in its envs, see
+        # https://github.com/conda/conda/issues/2764
+        env_prefix = os.environ.get("VIRTUAL_ENV", os.environ.get("CONDA_PREFIX"))
+        conda_env_name = os.environ.get("CONDA_DEFAULT_ENV")
+        # It's probably not a good idea to pollute Conda's global "base" env, since
+        # most users have it activated all the time.
+        in_venv = env_prefix is not None and conda_env_name != "base"
```

Comment:
@maksbotan, @stephsamson it is useful to allow poetry install into "base" environment when we create Docker image because there no need to create multiple Conda environments inside the container. 
Otherwise, the developer should be aware that Poetry will ignore "base" environment. It would be great to output a warning/notice message about this.